vcpkg.json

```
@@ -29,6 +29,39 @@
         "zstd"
       ]
     },
+    {
+      "name": "dawn",
+      "platform": "android",
+      "default-features": false,
+      "features": [
+        "vulkan"
+      ]
+    },
+    {
+      "name": "dawn",
+      "platform": "linux | freebsd | openbsd",
```

```suggestion
      "platform": "linux | bsd",
```

Makes this a tiny bit cleaner